components/Launcher.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <v-container class="space-around">
+    <v-row align-content="center" class="justify-center">
+      <v-col v-if="((!captcha_validated) && ($config.NODE_ENV === 'production'))" cols="12" class="text-center">
+        <v-card class="pb-5" elevation="5">
+          <v-card-title class="justify-center">
+            Please confirm that you are not a robot
+          </v-card-title>
+          <v-card-text class="text-center">
+            Please confirm that you're not a robot before launching the app
+          </v-card-text>
+        </v-card>
+        <recaptcha class="d-flex justify-space-around" />
```

Comment:
We may need more spacing between all these elements. What about using the column layout and adding each element in a v-col ?